packages/ui-new/src/utils/ThemeConstantsUtil.ts

```diff
@@ -0,0 +1,308 @@
+
+    /* Text colors */
+    textAccentPrimary: '#0988F0',
+    textAccentCertified: '#C7B994',
+
+    textWalletKit: '#FFB800',
+    textAppKit: '#FF573B',
+    textCloud: '#0988F0',
+    textDocumentation: '#008847',
+
+    textSuccess: '#30A46B',
+    textError: '#DF4A34',
+    textWarning: '#F3A13F',
+
+    /* Border colors */
+    borderAccentPrimary: '#0988F0',
+    borderSecondary: '#C7B994',
+
+    borderSuccess: '#30A46B',
+    borderError: '#DF4A34',
+    borderWarning: '#F3A13F',
+
+    /* Foreground colors */
+    foregroundAccent010: 'rgba(9, 136, 240, 0.1)',
+    foregroundAccent020: 'rgba(9, 136, 240, 0.2)',
+    foregroundAccent040: 'rgba(9, 136, 240, 0.4)',
+    foregroundAccent060: 'rgba(9, 136, 240, 0.6)',
+
+    foregroundSecondary020: 'rgba(199, 185, 148, 0.2)',
+    foregroundSecondary040: 'rgba(199, 185, 148, 0.4)',
+    foregroundSecondary060: 'rgba(199, 185, 148, 0.6)',
+
+    /* Icon colors */
+    iconAccentPrimary: '#0988F0',
+    iconAccentCertified: '#C7B994',
+
+    iconSuccess: '#0988F0',
+    iconError: '#DF4A34',
+    iconWarning: '#F3A13F'
+  },
+  dark: {
+    /* Background colors */
+    backgroundPrimary: '#202020',
+    backgroundInvert: '#FFFFFF',
+
+    /* Text colors */
+    textPrimary: '#FFFFFF',
+    textSecondary: '#9A9A9A',
+    textTertiary: '#BBBBBB',
+    textInvert: '#202020',
+
+    /* Border colors */
+    borderPrimary: '#2A2A2A',
+    borderSecondary: '#4F4F4F',
+
+    /* Foreground colors */
+    foregroundPrimary: '#2A2A2A',
+    foregroundSecondary: '#363636',
+    foregroundTertiary: '#4F4F4F',
+
+    /* Icon colors */
+    iconDefault: '#9A9A9A',
+    iconInverse: '#FFFFFF'
+  },
+  light: {
+    /* Background colors */
+    backgroundPrimary: '#202020',
+    backgroundInvert: '#202020',
+
+    /* Text colors */
+    textPrimary: '#202020',
+    textSecondary: '#9A9A9A',
+    textTertiary: '#6C6C6C',
+    textInvert: '#FFFFFF',
+
+    /* Border colors */
+    borderPrimary: '#E9E9E9',
+    borderSecondary: '#D0D0D0',
+
+    /* Foreground colors */
+    foregroundPrimary: '#F3F3F3',
+    foregroundSecondary: '#E9E9E9',
+    foregroundTertiary: '#D0D0D0',
+
+    /* Icon colors */
+    iconDefault: '#9A9A9A',
+    iconInverse: '#202020'
+  }
+}
+
+export const radii = {
+  '1': '4px',
+  '6': '6px',
+  '2': '8px',
+  '10': '10px',
+  '3': '12px',
+  '4': '16px',
+  '8': '32px',
+  '16': '64px',
+  '20': '80px',
+  '32': '128px',
+  '64': '256px',
+  '128': '512px',
+  round: '9999px'
+}
+
+export const spacing = {
+  '0.5': '2px',
+  '1': '4px',
+  '2': '8px',
+  '3': '12px',
+  '4': '16px',
+  '5': '20px',
+  '6': '24px',
+  '7': '28px',
+  '8': '32px',
+  '9': '36px',
+  '10': '40px',
+  '12': '48px',
+  '14': '56px',
+  '16': '64px',
+  '20': '80px',
+  '32': '128px',
+  '64': '256px'
+}
+
+const fontFamily = {
+  regular: 'KHTeka',
+  mono: 'KHTekaMono'
+}
+
+const fontWeight = {
+  regular: '400',
+  medium: '500'
+}
+
+const textSizes = {
```

Comment:
Like the others, we could use singular name
```suggestion
const textSize = {
```